__tests__/__system__/api/methods/define/Define.urimap-client.system.test.ts

```diff
@@ -115,10 +121,12 @@ describe("CICS Define client URImap", () => {
         options.name = urimapName;
         options.path = "fake";
         options.host = "fake";
-        options.scheme = "http";
+        options.scheme = "https";
         options.csdGroup = csdGroup;
         options.enable = enable;
         options.regionName = regionName;
+        options.authenticate = "BASIC";
+        options.certificate = "CICS";
```

Comment:
Certificate name still hardcoded here

Reply:
Thanks for pointing it out. It has been fixed.